Python/bltinmodule.c

```diff
@@ -252,13 +243,13 @@ builtin___build_class__(PyObject *self, PyObject *const *args, Py_ssize_t nargs,
     }
 error:
     Py_XDECREF(cell);
-    Py_DECREF(ns);
```

Comment:
Just above this at https://github.com/python/cpython/blob/main/Python/bltinmodule.c#L245,
there seems to be a refleak at the error path too? I'm not too sure, but doesn't it need ``Py_DECREF(cell_cls)`` too? ``PyCell_GET`` returns a new reference.